quickproject.lisp

```diff
@@ -29,6 +27,7 @@ in the pathname-directory list. E.g. returns \"awesome-project\" for
   (first (last (pathname-directory pathname))))
 
 (defun current-year ()
+  "Read the system year."
```

Comment:
This should be "Returns the current year." But at that point, what's the point of a docstring? The name is self-documenting.